tasks/scripts/containerd-integration

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+set -e -u -x
+
+export GOPATH=$PWD/gopath
+export PATH=$GOPATH/bin:$PATH
+
+source ci/tasks/scripts/docker-helpers.sh
+
+main() {
+  sanitize_cgroups
+  start_containerd
+  run_test
+}
+
+start_containerd() {
+  containerd >/tmp/containerd.log 2>&1 &
```

Comment:
we should be providing our own configuration here to ensure that we don't get any collisions with `docker`'s `runc`, etc.

Reply:
https://github.com/concourse/ci/pull/253/commits/b6f26951005f1917e9ce22f0066586f2919afbfb